docs/adr/002-api-first.md

```diff
@@ -0,0 +1,41 @@
+# ADR 002 - API first
+
+Date: 26-11-2024
+
+## Status
+
+Accepted
+
+Written years after the decision was made.
+
+## Context
+
+The "API first" approach is to enforce that all interactions with the system must go through the API.
+See more about the "API first" approach [here](https://swagger.io/resources/articles/adopting-an-api-first-approach/).
+
+The previous version of OS2Display was used without the admin module in some contexts.
+We want to support other uses than the standard OS2Display setup.
+
+By adopting the API first approach it will be possible to replace clients without rewriting the entire application.
+This will make the system more future-proof.
+
+[OpenAPI](https://www.openapis.org/) is a standard for describing an API interface.
+
```

Comment:
As I understand it one of the great benefits of the API first approach also is that it is possible that content can be delivered from another admin/contentplatform than the OS2Display admin - maybe that is worth noting here